CHANGELOG.md

```diff
@@ -4,6 +4,10 @@
 - Debug from .csproj and .sln [#5876](https://github.com/dotnet/vscode-csharp/issues/5876)
 
 # Latest
+* Update Razor to 9.0.0-preview.24311.4 (PR: [#7225](https://github.com/dotnet/vscode-csharp/pull/7225))
```

Comment:
This needs to be in a new section, with the current latest renamed to 2.34.10 (it hasn't been updated since the prerelease)